components/server/src/orgs/organization-service.ts

```diff
@@ -187,6 +193,12 @@ export class OrganizationService {
 
                 await db.deleteTeam(orgId);
 
+                // todo: look into introducing an isStripe() method to simplify this here and other places
```

Comment:
`this.usageService.isBillingStrategyStripe` would work for me.

Small nit for the future: leaving small comments like this, though well-indended, are not super helpful because:
 - you will never get back to this line and say "ah, yes, let's clean this up now"
 - if you have time to clean this up later, you can also keep track of this in other places (issue, personal doc, etc.)
 
 Another nit: "How we work" says we should do `TODO(<3 letter acronym>)` :wink:

Reply:
Thanks for this, I agree the comment it not very useful. I'll update it.

Reply:
Backlogged EXP-1854 to track this.